include/EASTL/sort.h

```diff
@@ -191,7 +191,14 @@ namespace eastl
 			++result;
 		}
 
-		return eastl::copy(first2, last2, eastl::copy(first1, last1, result));
+		if (first1 == last1)
```

Comment:
Add an additional comment here so the code doesn't get reverted under future maintenance.

Reply:
Added!